kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/ProcessServicesClient.java

```diff
@@ -93,5 +94,11 @@
 
     List<WorkItemInstance> getWorkItemByProcessInstance(String containerId, Long processInstanceId);
 
+    List<NodeInstance> findActiveNodeInstances(String containerId, Long processInstanceId, Integer page, Integer pageSize);
```

Comment:
main reason is to have container id given so router can properly route it to correct server. But that's mainly relevant for router use case for simple server setup using the one from query client is enough because there won't be duplicates of the ids and there is no need to distinguish between containers.